From 1a9e889f16202593e2c583a0beb8a0ddc74b1e48 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Kirill=20M=C3=BCller?= <kirill@cynkra.com>
Date: Thu, 15 Oct 2026 14:05:38 +0000
Subject: [PATCH] =?UTF-8?q?Use=20https=20for=20Szabolcs=20Horv=C3=A1t's=20?=
 =?UTF-8?q?homepage?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The R package has linked this homepage over https;
the patch retires itself once upstream switches too.
---
 src/vendor/cigraph/.all-contributorsrc | 2 +-
 src/vendor/cigraph/CONTRIBUTORS.md     | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/vendor/cigraph/.all-contributorsrc b/src/vendor/cigraph/.all-contributorsrc
index 7da6c0da96..02f1940086 100644
--- a/src/vendor/cigraph/.all-contributorsrc
+++ b/src/vendor/cigraph/.all-contributorsrc
@@ -32,7 +32,7 @@
       "login": "szhorvat",
       "name": "Szabolcs Horvát",
       "avatar_url": "https://avatars.githubusercontent.com/u/1212871?v=4",
-      "profile": "http://szhorvat.net/",
+      "profile": "https://szhorvat.net/",
       "contributions": [
         "code"
       ]
diff --git a/src/vendor/cigraph/CONTRIBUTORS.md b/src/vendor/cigraph/CONTRIBUTORS.md
index faec64de7a..d53bdaad4f 100644
--- a/src/vendor/cigraph/CONTRIBUTORS.md
+++ b/src/vendor/cigraph/CONTRIBUTORS.md
@@ -10,7 +10,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <tr>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/gaborcsardi"><img src="https://avatars.githubusercontent.com/u/660288?v=4?s=100" width="100px;" alt="Gábor Csárdi"/><br /><sub><b>Gábor Csárdi</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=gaborcsardi" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://collmot.com/"><img src="https://avatars.githubusercontent.com/u/195637?v=4?s=100" width="100px;" alt="Tamás Nepusz"/><br /><sub><b>Tamás Nepusz</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=ntamas" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="http://szhorvat.net/"><img src="https://avatars.githubusercontent.com/u/1212871?v=4?s=100" width="100px;" alt="Szabolcs Horvát"/><br /><sub><b>Szabolcs Horvát</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=szhorvat" title="Code">💻</a></td>
+      <td align="center" valign="top" width="14.28%"><a href="https://szhorvat.net/"><img src="https://avatars.githubusercontent.com/u/1212871?v=4?s=100" width="100px;" alt="Szabolcs Horvát"/><br /><sub><b>Szabolcs Horvát</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=szhorvat" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="http://www.traag.net/"><img src="https://avatars.githubusercontent.com/u/6057804?v=4?s=100" width="100px;" alt="Vincent Traag"/><br /><sub><b>Vincent Traag</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=vtraag" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/GroteGnoom"><img src="https://avatars.githubusercontent.com/u/8137208?v=4?s=100" width="100px;" alt="GroteGnoom"/><br /><sub><b>GroteGnoom</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=GroteGnoom" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://fabilab.org/"><img src="https://avatars.githubusercontent.com/u/1200640?v=4?s=100" width="100px;" alt="Fabio Zanini"/><br /><sub><b>Fabio Zanini</b></sub></a><br /><a href="https://github.com/igraph/igraph/commits?author=iosonofabio" title="Code">💻</a></td>
-- 
2.55.0

